app/helpers/fuzzy_time.rb

```diff
@@ -22,9 +23,9 @@ def ago(timestamp)
                      when less_than(23.5 * hours)
                        "about #{(diff / (1 * hours)).round} hours ago"
                      end
-          "<span data-toggle='tooltip' title='#{timestamp.strftime('%e %B %Y at %H:%M %Z')}'>#{response}</span>"
+          "<span data-toggle='tooltip' data-title='#{formatted_date}'>#{response}</span>"
         else
-          timestamp.strftime("%e %B %Y at %H:%M %Z")
+          "<span class='localize-time'>#{formatted_date}</span>"
         end
```

Comment:
Some questions due to my unfamiliarity with the javascript portions:

Is this `diff` condition still relevant? Does the javascript not handle times in the last 24 hours?

Does the `data-toggle='tooltip' data-title='#{formatted_date}'` mean that you see the real timestamp if you mouseover?

Does the new javascript allow you to see both the localized version and the real timestamp?